akka_microservices/ibex35Microservice/src/main/scala/com/tapiporla/microservices/retrievers/indices/stock/model/StockHistoric.scala

```diff
@@ -0,0 +1,58 @@
+package com.tapiporla.microservices.retrievers.indices.stock.model
+
+import com.github.nscala_time.time.Imports._
+import com.sksamuel.elastic4s.Hit
+import com.tapiporla.microservices.retrievers.common.TapiporlaConfig
+import com.tapiporla.microservices.retrievers.common.model.{DataInputExtractable, ElasticDocumentInsertable}
+import com.tapiporla.microservices.retrievers.common.stats.StatsGenerator.StatDataInput
+import com.tapiporla.microservices.retrievers.indices.stock.dao.StockESDAO
+
+object StockHistoric {
+  def fromMap(map: Map[String,String]) = {
```

Comment:
Seriously, I would call it ```values``` or whatever XD

Reply:
jajajajjajaja XD ok